common/spec/dependabot/notices_spec.rb

```diff
@@ -96,86 +114,6 @@ def initialize(name:, version:, deprecated_versions: [], unsupported_versions: [
     end
   end
 
-  describe ".generate_support_notice" do
```

Comment:
Not used generate support notice method are removed after finding out that we don't need to create notice. Instead we are using general error flow to raise unsupported error.